app/code/Magebit/Faq1/Controller/Adminhtml/Question/MassDelete.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Controller\Adminhtml\Question;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\App\Action\Context;
+use Magento\Ui\Component\MassAction\Filter;
+use Magento\Backend\App\Action;
+use Magebit\Faq1\Model\ResourceModel\Question\CollectionFactory;
+
+/**
+ * Class that let's us perform MassDelete action to multiple rows
+ */
+class MassDelete extends Action
+{
+    /**
+     * @var Filter
+     */
+    protected Filter $filter;
+
+    /**
+     * @var CollectionFactory
+     */
+    protected CollectionFactory $collectionFactory;
+
+
+    /**
+     * @param Context $context
+     * @param Filter $filter
+     * @param CollectionFactory $collectionFactory
+     */
+    public function __construct(Context $context, Filter $filter, CollectionFactory $collectionFactory)
+    {
+        parent::__construct($context);
+        $this->filter = $filter;
+        $this->collectionFactory = $collectionFactory;
+    }
+    /**
+     * Execute action
+     *
+     * @return \Magento\Backend\Model\View\Result\Redirect
+     * @throws \Magento\Framework\Exception\LocalizedException|\Exception
+     */
+    public function execute()
+    {
+        $collection = $this->filter->getCollection($this->collectionFactory->create());
+        $collectionSize = $collection->getSize();
+        $questionDeleted=0;
+        foreach ($collection as $item) {
+            $deleteItem = $this->_objectManager->get('Magebit\Faq1\Model\Question')->load($item->getId());
+            $deleteItem->delete();
```

Comment:
Do not use ObjectManager, it shouldn't be used ever. Use your QuestionRepository to delete the item, you can pass the delete function the item, which would be the Question Model/Interface type of object